examples/tests/sdk_tests/bucket/delete.w

```diff
@@ -0,0 +1,22 @@
+bring cloud;
+
+let b = new cloud.Bucket();
+
+test "delete" {
+  let jsonObj1 = Json { key1: "value1" };
+
+  b.putJson("file1.json", jsonObj1);
+  b.delete("file1.txt");
+
+  assert(b.list().length == 1);
+
+  b.delete("file1.json");
+  b.delete("file1.json");
```

Comment:
can you please test with the `mustExist` argument too? (and deleting a file uploaded via `addObject`?)